patterns/page-newsletter-subscribe.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Title: Newsletter Subscribe
+ * Slug: twentytwentyfour/page-newsletter-subscribe
+ * Categories: call-to-action
```

Comment:
looks like this is incompatible with postTypes: page, which one do we prefer?

Reply:
Not following? I added it to support the page post type.

Reply:
I don't think we want it in the starter pattern modal for when you're creating a new post.

Reply:
if it's only for post types page this pattern can't be inserted on a post, even if we use the pattern block (I'm guessing that's what we want but wanted to make it clear)

Reply:
Ah right. Yes I think that's a fair trade (to include it in the starter patterns). As it's a full viewport page design I'm cool with it. We can add alternates later once the patterns all fit in and look ace.